app/models/appointment.rb

```diff
@@ -1,14 +1,16 @@
 class Appointment < ActiveRecord::Base
+  COMPLETE = 'Complete'.freeze
```

Comment:
It is also used in `lib/importers/cita/record.rb` which is the reason I replaced it. :)